src/bit_vec.rs

```diff
@@ -15,17 +15,13 @@
 //! `BitVec` specific serialization.
 
 use bitvec::{
-	vec::BitVec, store::BitStore, order::BitOrder, slice::BitSlice, boxed::BitBox,
+	vec::BitVec, store::BitStore, order::BitOrder, slice::BitSlice, boxed::BitBox, view::BitView,
 };
 use crate::{
-	EncodeLike, Encode, Decode, Input, Output, Error, Compact,
-	codec::{decode_vec_with_len, encode_slice_no_len},
+	EncodeLike, Encode, Decode, Input, Output, Error, Compact, codec::decode_vec_with_len,
 };
 
-impl<O: BitOrder, T: BitStore> Encode for BitSlice<T, O>
-	where
-		T::Mem: Encode
-{
```

Comment:
This technically is semver breaking, but I would go this route. `BitVec` already required the same bound.